File: backend/src/main/java/org/luxons/sevenwonders/SevenWonders.java
package org.luxons.sevenwonders;

import org.hildan.livedoc.spring.boot.starter.EnableJSONDoc;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@EnableJSONDoc
public class SevenWonders {

    public static void main(String[] args) {
        SpringApplication.run(SevenWonders.class, args);
    }
}

File: sw-server/src/main/kotlin/org/luxons/sevenwonders/SevenWonders.kt
package org.luxons.sevenwonders

import org.hildan.livedoc.spring.boot.starter.EnableJSONDoc
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableJSONDoc
class SevenWonders

fun main(args: Array<String>) {
    runApplication<SevenWonders>(*args)
}
